product-development/README.md

```diff
@@ -104,57 +117,102 @@ Example:
+  even if other people on the team disagree. Consensus is a bad way to make
+  decisions, and we shouldn't require consensus on an idea to move forward.
+  
+- It's anxiety-producing and not helpful when people are constantly switching
+  back and forth between execution and prioritization. We don't want each DRI
+  to have to do a prioritization exercise every week when they finish their epic.
+  
+So we're trying a new system.
+
+Each Tilter is responsible for reading the backlog of epics. They don't need to
+understand every epic. But they should understand enough to be able to
+affirmatively articulate what epics in the backlog they want to work on and why.
+
+Every two weeks, Nick will talk to you your impressions of the backlog, and try
+to get a sense of what you think is important and want to work on.
+
+Nick will assign epics near the top of the backlog, trying to balance
+preferences and current priority order.
 
-<img src="images/above-the-line.png" height="150" />
+If there's an epic that's high in priority order but no one wants to work on,
+then Nick should dig more into why. Do we need to do more research upfront?  Is
+the epic defined incorrectly? Is it misaligned with what we're currently
+focusing on?
 
-## Reducing scope
-A given epic may have a fairly broad initial business problem. The engineer(s) should carve out and define a smaller problem, and spec out a solution that can be accomplished within a target of **_7 business days_** (counted from when the epic moves into `In Progress`, more below on states). If the solution cannot be finished within 7 business days, keep reducing scope. The epic description should be updated with the smaller scope and acceptance criteria, and the engineer(s) should create additional epics and/or stories to capture the future work. Alternatively, the engineer(s) may create a separate epic with the smaller scope and start work on that one instead (putting the original epic back into the backlog).
+There may be rare cases where someone gets assigned to an epic that they
+don't want to work on, but in those cases we should explicitly call out
+that this is happening and why.
+
+At all times, every Tilter should have 1-2 epics assigned to them that haven't
+been started yet. The epics are their "Next Up" list.  This isn't final - they
+may get reassigned. But we bias towards keeping the current assignment so that
+Tilters know what's coming next.
+
+### Starting an Epic
+
+When you're ready to start an epic, it should be clear which epic to start work
+on -- it's the highest-priority backlog epic assigned to you.
+
+Move it into the `In Progress` state.
+
+## How an Epic Progresses
+
+A given epic may have a fairly broad initial business problem. The engineer(s) should carve out and define a smaller problem, and spec out a solution that can be accomplished within a target of **_7 business days_** (counted from when the epic moves into `In Progress`, more below on states). 
```

Comment:
Is 7 business days Monday to Monday or Monday to Wednesday?

Reply:
i think monday to wednesday (this isn't new, just moved)